src/Asv.Mavlink.Test/Microservices/Ftp/FtpExComplexTest.cs

```diff
@@ -257,13 +257,15 @@ private static MockFileSystem SetUpFileSystem(string root)
     public async Task DownloadFile_ToStream_Cancel_Throws(int size, byte partSize)
     {
         // Arrange
-        var server = Server;
+        _ = Server;
         await _cts.CancelAsync();
         const string fileName = "test.txt";
         Debug.Assert(_fileSystem != null, nameof(_fileSystem) + " != null");
         var filePath = _fileSystem.Path.Combine(_serverExConfig.RootDirectory, fileName);
-        var originContent = new byte[size];
-        new Random().NextBytes(originContent);
+        var originContent = 
+            Random.Shared.GetItems(
+                Enumerable.Range(0, 255).Select(i => (byte)i).ToArray(), 
+                size);
```

Comment:
Все похожие случаи лучше так сделать, но это 50/50 nit
```suggestion
var originContent = 
            Random.Shared.GetItems(
                Enumerable.Range(0, 255).Select(i => (byte)i).ToArray(), 
                size);
        _fileSystem.AddFile(filePath, new MockFileData(originContent));

        using var streamToSave = new MemoryStream(size);
        
        // Act
        var task = Client.DownloadFile(
            fileName, 
            streamToSave, 
            new CallbackProgress<double>(_ => { }), partSize,
            _cts.Token
        );
        
        // Assert
        await Assert.ThrowsAsync<OperationCanceledException>(async () =>
        {
            await task;
        });
```